pkg/utils/util.go

```diff
@@ -0,0 +1,22 @@
+package utils
+
+import (
+	"bytes"
+
+	"github.com/celestiaorg/celestia-app/pkg/appconsts"
+	"github.com/celestiaorg/nmt/namespace"
+	tmrand "github.com/tendermint/tendermint/libs/rand"
+)
+
+func RandomValidNamespace() namespace.ID {
+	for {
+		ns := tmrand.Bytes(8)
```

Comment:
[Blocking][Question]
This function seems to create a valid namespace that is a message. However, some of the other deleted functions:
```golang
func randomValidNamespace() namespace.ID {
	for {
		s := tmrand.Bytes(8)
		if bytes.Compare(s, appconsts.MaxReservedNamespace) > 0 {
			return s
		}
	}
}
```
Seem to define a valid namespace as smaller than the max reserved namespace.
Is this alright? 
Do we need to generate messages namespaces in some tests while in others we need any namespace that is `<= MaxReservedNamespace`?

Reply:
The one that was kept also validated that `bytes.Compare(ns, appconsts.MaxReservedNamespace) <= 0` and I believe in a better way.
Also, the idea of keeping this specific function, came from the original issue https://github.com/celestiaorg/celestia-app/issues/836 where @rootulp mentioned:
> The last function definition is the most preferred because it accounts for the parity and tail padding namespaces. 

So, this one only should be fine.

Reply:
Ah, sounds good :+1: